packages/client/src/components/NewArrivalsCarousel/NewArrivalsCarousel.jsx

```diff
@@ -0,0 +1,208 @@
+import React, { useRef } from 'react';
+import Slider from 'react-slick';
+import {
+  Box,
+  Button,
+  IconButton,
+  Typography,
+  Link,
+  Container,
+} from '@mui/material';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import styled from '@emotion/styled';
+import { useNewArrivals } from './useNewArrivals';
+import './styles.css';
+
+const settings = {
+  className: 'center-slide',
+  centerMode: true,
+  infinite: true,
+  centerPadding: '370px',
+  slidesToShow: 1,
+  swipeToSlide: true,
+  slidesToScroll: 1,
+  dots: true,
+  arrows: false,
+  autoplay: false,
+  speed: 500,
+  autoplaySpeed: 3000,
+  cssEase: 'linear',
+};
+
+export const NewArrivalsCarousel = () => {
+  const [newArrivals] = useNewArrivals();
+  const sliderRef = useRef();
+
+  const handleSliderPrev = () => {
+    sliderRef.current?.slickPrev();
+  };
+
+  const handleSliderNext = () => {
+    sliderRef.current?.slickNext();
+  };
+
+  return (
+    <MainContainer maxWidth="false">
```

Comment:
```suggestion
    <Container sx={{....}}>
```
I would recommend using the inline styling with sx instead of recreating the component with styled.